### PR TITLE
add title for options to see full display value on hover

### PR DESCRIPTION
**Pull request by MichelML**
very small change just to be able to have a preview of the full display value on hover of an option or selected option, see attached images

### CODE REVIEW

**Comment by MichelML** [comment]
screen shot 2017-08-01 at 10 27 18 am
screen shot 2017-08-01 at 10 27 18 am 2

**Comment by MichelML** [comment]
screen shot 2017-08-01 at 10 27 14 am 2

**Comment by MichelML** [comment]
screen shot 2017-08-01 at 10 27 18 am 2

File: src/components/dropdownSearch/DropdownSearch.tsx [hunk 1]
```diff
@@ -96,7 +97,7 @@ export class DropdownSearch extends React.Component<IDropdownSearchProps, void>
           : value;
 
         return <li key={option.value}
-          className={liClasses}>
```
Comment (pochretien) [inline src/components/dropdownSearch/DropdownSearch.tsx]: move title on the next line

Comment (MichelML) [inline src/components/dropdownSearch/DropdownSearch.tsx]: done

**Pull request merged**